src/content/tools/doc-comment-references.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Documentation comment references
+short-title: Comment references
+description: Learn about doc comment references and their syntax.
+---
+
+Doc comments can contain references to various code elements. A library member
+or class member, etc. can be referenced by wrapping its name in square brackets
```

Comment:
We try to [avoid use of etc](https://developers.google.com/style/word-list#etc) if possible.

```suggestion
Doc comments can contain references to various code elements.
Elements, such as functions and classes, can be referenced by wrapping their name in square brackets
```

Reply:
Done